src/linux/backend-v4l2.cpp

```diff
@@ -732,7 +794,7 @@ namespace librealsense
             usb_spec usb_specification(usb_undefined);
             std::string bus_info, card;
 
-            auto dev_name = "/dev/" + name;
+            auto dev_name = "/dev/" + get_devname_from_video_path(video_path);
```

Comment:
This procedure getting MIPI cameras.
This will not work in case we use links for video nodes.
See line 882

Reply:
Yes I see now the MIPI functions retrieve their devices' info from ioctl not sysfs correct? I will remove the changes I made to those functions, they should just work in containers already, although I don't have hardware to test.